gcpy/benchmark/modules/run_1yr_tt_benchmark.py

```diff
@@ -466,8 +463,21 @@ def run_benchmark(config, bmk_year_ref, bmk_year_dev):
         # ==================================================================
         if config["options"]["outputs"]["rnpbbe_budget"]:
             print("\n%%% Creating GCC vs. GCC radionuclides budget table %%%")
-            ttbdg.transport_tracers_budgets(
-                config["data"]["dev"]["gcc"]["dir"],
+
+            # Ref
+            transport_tracers_budgets(
+                config["data"]["ref"]["gcc"]["version"],
+                gcc_vs_gcc_refdir,
+                gcc_vs_gcc_refrstdir,
+                int(bmk_year_ref),
+                dst=gcc_vs_gcc_tablesdir,
+                overwrite=True,
+                spcdb_dir=spcdb_dir,
```

Comment:
In the future, it would be nice to print Ref and Dev to the same file so the values could be compared side-by-side. We do this for other benchmark tables.  It's a little bit tricky here because the budget and STE tables have three columns (210Pb, 7Be, 10Be), but each of those could be include into two subcolumns.

This comment also goes for other tables below.

Reply:
@msulprizio, agreed.  I'll make a separate feature request for this so we don't lose sight of it.

Reply:
@msulprizio: This is now a feature request in #308